### DISCUSSION TITLE
Lazy Components Removing Attributes from Models Passed as Props

### DISCUSSION THREAD

**Discussion by ZebTheWizard** [livewire/livewire#6420] [1. Help]
I am looping over posts and passing the like count to a like button component. The `liked` and `likeCount` attributes are unavailable when the component is lazy.

```php
$posts = Post::withCount([
  'likers as likeCount',
  'likers as liked' => fn($q) => $q->where('id', Auth::id())
])->get();
```

```blade
@foreach ($posts as $post)
  <div>
    {{ $post->content}}
    <livewire:likeButton :post="$post" lazy />
  </div>
@endforeach
```

If I remove the lazy attribute, then all the data is available.

**Comment by PhiloNL** [comment]
What does the `likeButton` component look like?
Given you already pass the like count, I'm curious why you are adding the lazy attribute.
